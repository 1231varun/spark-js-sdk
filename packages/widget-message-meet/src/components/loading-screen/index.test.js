import React from 'react';
import renderer from 'react-test-renderer';

import LoadingScreen from '.';

describe(`LoadingScreen component`, () => {

  const component = renderer.create(
    <LoadingScreen />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
