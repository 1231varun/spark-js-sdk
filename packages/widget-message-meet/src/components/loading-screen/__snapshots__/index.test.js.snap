exports[`LoadingScreen component renders properly 1`] = `
<div
  className="loading">
  <div
    className="spark-logo">
    <div
      className="spark-logo">
      <img
        role="presentation"
        src={Object {}} />
    </div>
  </div>
  <span>
    Connecting to Spark
  </span>
  <div
    className="spinner-container">
    <div
      className="spinner" />
  </div>
</div>
`;
